2200091/1225.py

```diff
@@ -0,0 +1,44 @@
+import sys
+from collections import deque
+sys.stdin = open("input.txt")
+input = sys.stdin.readline
+
+for _ in range(10):
+    test = int(input())
+    numbers = list(map(int, input().split()))
+    # 앞뒤에서 삽입 삭제가 필요하니까 deque 사용
+    que = deque()
+
+    # 입력으로 들어온 수를 deque에 저장
+    for num in numbers:
+        que.append(num)
```

Comment:
깔끔한 풀이네요! 👍
```suggestion
    que = deque(map(int, input().split()))
```
`list`를 거치지 않고 바로 `deque`을 생성하셔도 좋을 것 같습니다 😄